lib/cocoapods-core/specification/json.rb

```diff
@@ -32,11 +31,8 @@ def to_hash
         all_testspecs = specs_by_type[:test] || []
         all_subspecs = specs_by_type[:library] || []
 
-        hash.delete('testspecs')
```

Comment:
I think we still want to do testspec, appspecs, and subspecs since they're handled specially in `from_hash`?